presto-main/src/main/java/com/facebook/presto/operator/aggregation/AggregationUtils.java

```diff
@@ -124,6 +143,37 @@ public static void mergeVarianceState(VarianceState state, VarianceState otherSt
         state.setMean(newMean);
     }
 
+    public static void mergeCentralMomentsState(CentralMomentsState state, CentralMomentsState otherState)
+    {
+        long na = state.getCount();
+        long nb = otherState.getCount();
+
+        checkArgument(nb >= 0, "count is negative");
+        if (nb == 0) {
+            return;
+        }
+
+        double m1a = state.getM1();
+        double m2a = state.getM2();
+        double m3a = state.getM3();
+        double m1b = otherState.getM1();
+        double m2b = otherState.getM2();
+        double m3b = otherState.getM3();
+        long n = na + nb;
+        long n2 = n * n;
+        double delta = m1b - m1a;
+        double delta2 = delta * delta;
+        double delta4 = delta2 * delta2;
+
+        state.setCount(n);
+        state.setM1((na * m1a + nb * m1b) / n);
+        state.setM2(m2a + m2b + (delta2 * na * nb) / n);
+        state.setM3(m3a + m3b + Math.pow(delta, 3) * na * nb * (na - nb) / n2 + 3 * delta * (na * m2b - nb * m2a) / n);
+        double m4 = state.getM4() + otherState.getM4() + delta4 * na * nb * (na * na - na * nb + nb * nb) / Math.pow(n, 3);
```

Comment:
Similar issue about strength reduction here with `Math.pow(n, 3)`, which could be replaced with `n * n * n` to ensure there's an optimization and maximize comparability with the original source code.